flocker/node/functional/test_docker.py

```diff
@@ -641,6 +644,28 @@ def test_restart_policy_on_failure_maximum_count(self):
         d.addCallback(self.assertIn, ("5", "6"))
         return d
 
+    def test_command_line(self):
+        """
+        A container with custom command line is run with those arguments.
+        """
+        directory = FilePath(self.mktemp())
+        directory.makedirs()
+        d = self.start_container(
+            random_name(), image_name=u"busybox",
```

Comment:
We decided that it does, in fact, have an entrypoint (the default one, `/bin/sh`).